Thiago-Pereira/1.iii/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Is your document in english or portuguese? I could see both languages!
Ps: If it is portuguese language we need to update the attribute lang to "pt-br"